topic_admin.go

```diff
@@ -0,0 +1,74 @@
+package kazoo
+
+import (
+	"errors"
+	"fmt"
+)
+
+var (
+	ErrTopicExists          = errors.New("Topic already exists")
+	ErrTopicMarkedForDelete = errors.New("Topic is already marked for deletion")
+)
+
+// CreateTopic creates a new kafka topic with the specified parameters and properties
```

Comment:
Looking at the implementation again as well as your implementation in the ruby library (https://github.com/wvanbergen/kazoo/blob/master/lib/kazoo/topic.rb), it appears to be synchronous.